react/components/EXPERIMENTAL_Table/DataTable/Rows.tsx

```diff
@@ -53,6 +53,9 @@ const Rows: FC<RowsProps> = ({
                     <CellPrefix.Checkbox
                       checked={isRowChecked}
                       partial={isRowPartiallyChecked}
+                      disabled={
+                        checkboxes.isDisabled && checkboxes.isDisabled(rowData)
```

Comment:
Shouldn't `checkboxes.isDisabled` be enough?

Reply:
yes, it should